src/components/modal/modal.html

```diff
@@ -0,0 +1,137 @@
+<!DOCTYPE html>
+<html lang="en">
```

Comment:
UX Suggestions:  
1. Alignment could be fixed.

![image](https://user-images.githubusercontent.com/91944618/154857707-158bf8a1-6932-41c9-85ec-3815b1fce05b.png)

2. There are two links for github page. 
![image](https://user-images.githubusercontent.com/91944618/154858044-8095368e-fa10-4198-8874-0748434216f0.png)